hibernate-entitymanager/src/main/java/org/hibernate/jpa/boot/internal/EntityManagerFactoryBuilderImpl.java

```diff
@@ -307,8 +316,12 @@ public boolean doExtendedEnhancement(CtClass classDescriptor) {
 	 */
 	private BootstrapServiceRegistry buildBootstrapServiceRegistry(
 			Map integrationSettings,
-			ClassLoader providedClassLoader) {
+			ClassLoader providedClassLoader, 
+			ClassLoaderService providedClassLoaderService) {
 		final BootstrapServiceRegistryBuilder bsrBuilder = new BootstrapServiceRegistryBuilder();
+		if (providedClassLoaderService != null) {
+			bsrBuilder.applyClassLoaderService(providedClassLoaderService);
```

Comment:
Answered my own question.  @kankt, see `BootstrapServiceRegistryBuilder#build`.  `providedClassLoaderService` and `providedClassLoaders` are mutually exclusive.  So, there's no reason to support both in the same method sig.